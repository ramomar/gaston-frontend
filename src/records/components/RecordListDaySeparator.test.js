import React from 'react';
import renderer from 'react-test-renderer';
import RecordListDaySeparator from './RecordListDaySeparator';
import { DateTime } from 'luxon';

it('renders correctly', () => {
  const component =
    <RecordListDaySeparator
      day={DateTime.fromISO('2019-12-30T02:58:24.948Z')}
      amountForDay={100.50} />;

  const tree = renderer.create(component).toJSON();

  expect(tree).toMatchSnapshot();
});
