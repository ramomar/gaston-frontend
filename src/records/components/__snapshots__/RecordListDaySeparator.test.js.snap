// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 itbAYy"
  style={
    Object {
      "position": "sticky",
      "top": 0,
    }
  }
>
  <span
    className="StyledText-sc-1sadyjn-0 bnqisW"
  >
    29 de December
  </span>
  <div
    className="StyledBox-sc-13pk1d4-0 cgqXer"
  >
    <span
      className="StyledText-sc-1sadyjn-0 hLzlgx"
      size="small"
    >
      -$
      100.50
    </span>
  </div>
</div>
`;
